src/apps/sorting-visualizer/components/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC, ReactNode } from 'react';
+import classes from "./tooltip.module.scss";
+
+interface TooltipProps {
+  text: string;
+  children: ReactNode;
+}
```

Comment:
interface to be moved to interfaces file and imported for keeping it simple for JS devs